openstack/clustering/v1/profiles/requests.go

```diff
@@ -40,3 +41,40 @@ func Create(client *gophercloud.ServiceClient, opts CreateOptsBuilder) (r Create
 	}
 	return
 }
+
+// ListOptsBuilder Builder.
+type ListOptsBuilder interface {
+	ToProfileListQuery() (string, error)
+}
+
+// ListOpts params
+type ListOpts struct {
+	GlobalProject bool   `q:"global_project"`
```

Comment:
One additional comment: similar to #889, if `false` is a valid value to pass, then this should be `*bool`.